src/components/CoffeeModal.tsx

```diff
@@ -63,7 +69,16 @@ const CoffeeModal = ({ product, isOpen, onClose }: { product: Coffee; isOpen: bo
         <IconOutlineButton icon={<AddIcon />} aria-label={"Increment"} onClick={increment} />
       </HStack>
       <div>
-        <ActionButton onClick={onClose} square leftIcon={<AddIcon />} size={"md"} px={8}>
+        <ActionButton
+          onClick={() => {
+            onAdd();
```

Comment:
we'll eventually need `price, size, grind, quantity` added here and maybe just name/image instead of the whole product object in cart.tsx, but there's been some changes in the modals/pages in the other PRs so this works for now